launcher/launcher.go

```diff
@@ -797,6 +804,43 @@ nextNetInstance:
 	}
 }
 
+func (launcher *Launcher) removeInstancesUID(instances []cloudprotocol.InstanceInfo) {
```

Comment:
Looks like this function removes instances from DB as well. Should we rename it to `removeInstances`?

Reply:
Agree, Done